Src/HALALMock/Services/Communication/FDCAN/FDCAN.cpp

```diff
@@ -43,40 +53,33 @@ void FDCAN::start(){
 	for( std::pair<uint8_t, FDCAN::Instance*> inst: FDCAN::registered_fdcan){
 		uint8_t id = inst.first;
 		FDCAN::Instance* instance = inst.second;
-		FDCAN::init(instance);
-
-		FDCAN_TxHeaderTypeDef header;
-		header.FDFormat = FDCAN_FD_CAN;
-		header.DataLength = instance->dlc;
-		header.TxFrameType = FDCAN_DATA_FRAME;
-		header.BitRateSwitch = FDCAN_BRS_ON;
-		header.ErrorStateIndicator = FDCAN_ESI_ACTIVE;
-		header.IdType = FDCAN_STANDARD_ID;
-		header.TxEventFifoControl = FDCAN_NO_TX_EVENTS;
-		header.MessageMarker = 0;
-		header.Identifier = 0x0;
-
-		instance->tx_header = header;
 
 		instance->rx_queue = queue<FDCAN::Packet>();
 		instance->tx_data = vector<uint8_t>();
 
-		if(HAL_FDCAN_Start(instance->hfdcan) != HAL_OK){
-			ErrorHandler("Error during FDCAN %d initialization.", instance->fdcan_number);
+		instance -> socket = socket(AF_NET,SOCK_STREAM,0);
```

Comment:
```suggestion
		instance -> socket = socket(AF_NET,SOCK_DGRAM,0);
```
otherwise it is TCP, and to do broadcast you need to use UDP